### PR TITLE
[cleanup] Switch to OkHttp from dumb http libraries

### PR DESCRIPTION
**Pull request by l1ving**
Signed-off-by: lv <~@l1v.in>

### CODE REVIEW

**Comment by l1ving** [comment]
Tested with shrunk jar in non-dev, works
image

**Review by Xiaro — CHANGES_REQUESTED** [review]

File: src/main/java/me/zeroeightsix/kami/KamiMod.java [hunk 1]
```diff
@@ -43,6 +44,8 @@
     @Mod.Instance
     public static KamiMod INSTANCE;
 
+    public static OkHttpClient httpClient = new OkHttpClient();
```
[the remark this answers is not in the document]
Comment (l1ving) [reply src/main/java/me/zeroeightsix/kami/KamiMod.java:47]: Why do I need a getter? I just want one instance of the client to use

[the remark this answers is not in the document]
Comment (Xiaro) [reply src/main/java/me/zeroeightsix/kami/KamiMod.java:47]: You are writing in Java not Kotlin

**Comment by l1ving** [comment]
Merging, can change it later if wanted

**Pull request merged**